cmd/alerts/list_alerts.go

```diff
@@ -0,0 +1,117 @@
+package alerts
+
+import (
+	"encoding/json"
+	"fmt"
+	"log"
+
+	"github.com/openshift/osdctl/cmd/alerts/silence"
+	"github.com/spf13/cobra"
+)
+
+type alertCmd struct {
+	clusterID  string
+	alertLevel string
+}
+
+type Labels struct {
+	Alertname string `json:"alertname"`
+	Severity  string `json:"severity"`
+}
+
+type Status struct {
+	State string `json:"state"`
+}
+
+type Annotations struct {
+	Summary string `json:"summary"`
+}
+
+type Alert struct {
+	Labels      Labels      `json:"labels"`
+	Status      Status      `json:"status"`
+	Annotations Annotations `json:"annotations"`
+}
+
+func NewCmdListAlerts() *cobra.Command {
+	alertCmd := &alertCmd{}
+	newCmd := &cobra.Command{
+		Use:               "list <cluster-id> --level [warning, critical, firing, pending, all]",
+		Short:             "List all alerts or based on severity",
+		Long:              `Checks the alerts for the cluster and print the list based on severity`,
+		Args:              cobra.ExactArgs(1),
+		DisableAutoGenTag: true,
+		Run: func(cmd *cobra.Command, args []string) {
+			alertCmd.clusterID = args[0]
+			ListAlerts(alertCmd)
+		},
+	}
+
+	newCmd.Flags().StringVarP(&alertCmd.alertLevel, "level", "l", "", "Alert level [warning, critical, firing, pending, all]")
+	return newCmd
+}
+
+func ListAlerts(cmd *alertCmd) {
+	var alerts []Alert
+	var levelCmd string
+
+	defer func() {
+		if err := recover(); err != nil {
+			log.Fatal("error : ", err)
+		}
+	}()
+
+	clusterID := cmd.clusterID
+	levelcmd := cmd.alertLevel
+
+	if levelcmd == "" {
+		fmt.Println("No alert level specified. Defaulting to 'all'")
```

Comment:
IMO debug style messages should be output to stderr instead of stdout.
We might also want to use some kind of verbosity level to not print in all cases.